computation-local/src/main/java/eu/itesla_project/computation/local/LocalComputationManager.java

```diff
@@ -15,6 +15,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.naming.OperationNotSupportedException;
```

Comment:
I didn't suggest the correct class this morning. Could you please change to UnsupportedOperationException